opm/autodiff/GeoProps.hpp

```diff
@@ -186,6 +192,11 @@ namespace Opm
                                      const double* perm,
                                      Vector &hTrans);
 
+        template <class Grid>
+        void minPvFillProps_(const Grid &grid,
```

Comment:
Minor nitpick (as usual): please use the standard C++ convention of attaching the `&` to the type not the variable name. It looks like taking the pointer to grid.